### PR TITLE
Hotfix/4/sc 8075 nbc ghost

### PR DESCRIPTION
**Pull request by ABialoskorski**
# Description
fix nbc ghost landing page

## Links to Tickets or other pull requests
- https://ticketsystem.schul-cloud.org/browse/SC-8075

## Changes
if when html from ghost is wrapped with 'pre' 'code' tags or not as it seems ghost is wrapping it randomly now

### CODE REVIEW

**Review by mwolowyk-cap — APPROVED** [review]

**Review by mwolowyk-cap — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "schulcloud-client",
-	"version": "25.3.0",
+	"version": "25.3.1",
```
Comment (virgilchiriac) [inline package.json:3]: same version in the other hotfix

Comment (ABialoskorski) [reply package.json:3]: I think that this PR will be merged first

Comment (ABialoskorski) [reply package.json:3]: I'll be in contact with Lukasz

**Review by virgilchiriac — DISMISSED** [review]

**Review by Metauriel — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Metauriel) [reply package.json]: thats fine, as the other hotfix is in the server

**Review by virgilchiriac — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Metauriel) [reply package.json]: keep in mind that https://github.com/hpi-schul-cloud/schulcloud-server/pull/2031#pullrequestreview-544953498 is on hte server, so the versions of those dont collide

**Review by Metauriel — APPROVED** [review]

**Pull request merged**